### PR TITLE
Extract country specific values into separate country specific text files

### PR DESCRIPTION
**Pull request by fossecode**

### CODE REVIEW

**Comment by fossecode** [comment]
Would be super if you guys could look through the PR thoroughly. Will be surprised if I haven't made any copy/paste mistakes.

File: app/views/privacy-statements/ar-privacy-statement.ejs [hunk 1]
```diff
@@ -70,14 +70,6 @@
...
-<p>
-  Contactanos en twitter:
-  <a
-    class="text-cvs-red hover:underline"
-    href="https://twitter.com/coronastatusAR"
-    target="_blank"
-    >@coronastatusAR</a
-  >.
-</p>
+<%- include('partials/contact') -%>
```
Comment (adriaandotcom) [inline app/views/privacy-statements/ar-privacy-statement.ejs:73]: Nice to move this away.

**Review by adriaandotcom — APPROVED** [review]
Checked the Dutch files and seems all good. Also checked some other files, but can't say I checked them all. Good work!

**Comment by sbocinec** [comment]
Wooow, massive PR!

Clicked through all the screens in the SK version, brought few minor fixes. Went through all other files hunting for whatever could be missed, but so far no luck. Great work!

**Review by sbocinec — APPROVED** [review]

**Comment by fossecode** [comment]
> Wooow, massive PR!
> 
> Clicked through all the screens in the SK version, brought few minor fixes. Went through all other files hunting for whatever could be missed, but so far no luck. Great work!

Yeah sorry for the big PR 😅Hard to keep it small when we have almost 30 different languages

**Pull request merged**